backend/api/serializers/GroupSerializer.py

```diff
@@ -0,0 +1,43 @@
+from api.models import Group
+from rest_framework import serializers
+from api.CustomField import BlankableIntegerField
+from django.core.validators import MinValueValidator
+
+
+class GroupSerializer(serializers.ModelSerializer):
+    name = serializers.CharField(read_only=True)
+    min_age = BlankableIntegerField(required=False, validators=[MinValueValidator(5)])
+    max_age = BlankableIntegerField(required=False, validators=[MinValueValidator(6)])
```

Comment:
I think it would make more sense to add the validation in the validate_age so the age could accept any age.